packages/backend/src/modules/tracked-txs/types/TrackedTxsConfig.ts

```diff
@@ -0,0 +1,48 @@
+import { Layer2TrackedTxUse } from '@l2beat/config'
+import { EthereumAddress, ProjectId, UnixTime } from '@l2beat/shared-pure'
+
+import { TrackedTxId } from './TrackedTxId'
+
+export const SHARP_SUBMISSION_ADDRESS = EthereumAddress(
+  '0x47312450B3Ac8b5b8e247a6bB6d523e7605bDb60',
+)
+export const SHARP_SUBMISSION_SELECTOR = '0x9b3b76cc'
+
+export type TrackedTxsConfig = {
+  entries: TrackedTxConfigEntry[]
+}
+
+export type TrackedTxConfigEntry =
+  | TrackedTxFunctionCallConfig
+  | TrackedTxTransferConfig
+  | TrackedTxSharpSubmissionConfig
+
+export interface TrackedTxUseWithId extends Layer2TrackedTxUse {
+  id: TrackedTxId
+}
+
+interface TrackedTxConfigBase {
+  projectId: ProjectId
+  uses: TrackedTxUseWithId[]
```

Comment:
I still feel like this should be named "usedBy"

Reply:
I will keep it for now, later we can change it